kotest-assertions/kotest-assertions-android/src/main/java/io/kotest/android/matchers/textview/matchers.kt

```diff
@@ -69,18 +75,3 @@ fun beAllCaps() = object : Matcher<TextView> {
         "TextView should not have AllCaps as transformation method"
     )
 }
-
-@RequiresApi(VERSION_CODES.JELLY_BEAN_MR1)
-infix fun TextView.shouldHaveTextAlignment(alignment: Int) = this should haveTextAlignment(alignment)
-
-@RequiresApi(VERSION_CODES.JELLY_BEAN_MR1)
-infix fun TextView.shouldNotHaveTextAlignment(alignment: Int) = this shouldNot haveTextAlignment(alignment)
-
-@RequiresApi(VERSION_CODES.JELLY_BEAN_MR1)
-fun haveTextAlignment(alignment: Int) = object : Matcher<TextView> {
-    override fun test(value: TextView) = MatcherResult(
-        value.textAlignment == alignment,
-        "TextView should have text alignment $alignment but was ${value.textAlignment}",
-        "TextView should not have text alignment $alignment, but had"
-    )
-}
```

Comment:
Removed cause of many issues with the alignment API, we might reintroduce later on